src/lib.rs

```diff
@@ -21,6 +21,7 @@
 //! ```
 
 extern crate chrono;
+extern crate deunicode;
```

Comment:
Not used

Reply:
I don't see edition 2018 in liquid's Cargo.toml, so it's still needed, isn't it?